### DISCUSSION TITLE
Does Dex providing a login page?

### DISCUSSION THREAD

**Discussion by dirkckau** [dexidp/dex#2280] [Q&A]
Hi, 

I am working on a solution to integrate a Windows AD authentication from the Kiali component in k8s. 
I've installed Dex in the k8s for my testing. 
However, I've been working for long but I still couldn't figure out the correct solution. 

Is it possible "Kiali -> Dex -> LDAP -> Kiali"? 

Does Dex providing a login page for the username and password input? 
How can I do it? 
If not, can I add an OAuth2 Proxy in between? 

Thanks.

**Comment by nabokihms** [comment]
Dex has a login page. It is enabled automatically for certain connectors that have no login page like LDAP, OpenStack Keystone, Atlassian Crowd, etc. See the following screenshot (prompt is configurable).

![Screenshot_20210921-074012](https://user-images.githubusercontent.com/32434187/134108565-3ab76500-38da-4ae4-afe4-9d81ae69d70d.png)

**Comment by dirkckau** [comment]
Hi, 

What is the relevant issuer URL? 
Or is there any parameter to direct to this page? 


Thanks.

**Reply by dirkckau** [reply]
error: "Error fetching OpenID provider metadata. "
detail: "cannot fetch OpenId Metadata (HTTP response status = 404 Not Found)"

Any idea about this error?
